docs/intro/dev_container.md

```diff
@@ -1 +1,77 @@
-# The dev container
+# The dev container-cli
+
+The dev-container cli takes devcontainer.json like the one in this worksapce and create and configure a dev container from it.
+This is handy for using the SDK without VS code UI.
+
+## Installation
+
+- docker installation
+
+Ensure to have node version greater 16 and not above 18
+
+```bash
+sudo apt-get update
+sudo apt-get install -y ca-certificates curl gnupg
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+export NODE_MAJOR=18
+echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | sudo tee /etc/apt/sources.list.d/nodesource.list
+sudo apt-get update
+sudo apt-get install nodejs -y
+```
+
+Then install devcontainers-cli itself
+
+```bash
+sudo apt install npm
+sudo npm install -g @devcontainers/cli
+```
+
+## Setup dev container
+
+- Building: `devcontainer up --workspace-folder ./ebcl_template/`
+- Execute a command inside `devcontainer exec --workspace-folder ./ebcl_template/ <command>`
+
+## Build an image
+
+Building an image can be done quite similar to the usual workflow in vsCode
+
+```bash
+devcontainer exec --workspace-folder ./ebcl_template/ bash
+(venv) ebcl@25b055e27967:/workspace$ cd images/amd64/appdev/qemu/ebcl_1.x_crinit
+(venv) ebcl@25b055e27967:/workspace/images/amd64/appdev/qemu/ebcl_1.x_crinit$ task
```

Comment:
```suggestion
(venv) ebcl@25b055e27967:/workspace/images/amd64/appdev/qemu/ebcl_1.x_crinit$ task build
```
Otherwise this runs qemu which doesn't really fit the description